modules/displaying-the-insights-status-in-the-web-console.adoc

```diff
@@ -20,4 +20,4 @@ Insights repeatedly analyzes your cluster and you can display the status of iden
 
 . Click *Insights* on the *Status* card.
 +
-The pop-up window lists potential issues grouped by priority. Click the individual categories or *View all in {product-title}* to display further details.
+The pop-up window lists potential issues grouped by priority. Click the individual categories or *View all recommendations in Insights Advisor* to display more details.
```

Comment:
Just a note: issues are grouped by risk. In other part of the text it referred as "prioritized by risk".

Reply:
good point, fixed it to 'grouped by risk' in both places.